web-client/src/presenter/computeds/myAccountHelper.ts

```diff
@@ -1,10 +1,12 @@
 import { ClientApplicationContext } from '@web-client/applicationContext';
 import { Get } from 'cerebral';
+import { state } from '@web-client/presenter/app.cerebral';
+
 export const myAccountHelper = (
   get: Get,
   applicationContext: ClientApplicationContext,
 ): any => {
-  const user = applicationContext.getCurrentUser();
+  const user = get(state.user);
```

Comment:
So you are telling me that we are storing the user in appContext and it is not accurate! Nice find.

Reply:
No, user is saved in appContext correctly but fetching from appContext was not re-triggering the computed.